### PR TITLE
Write a first draft of the January 2020 newsletter

### PR DESCRIPTION
**Pull request by rowanmanning**
I haven't added a "Broader Update" section yet, but wanted to get your thoughts on the "Top Three Things" and "Special Thanks". Let me know what you think! It might be better to publish tomorrow to give @chee a chance to look it over as well, but that will require someone else to do the conversion to email format and the sending.

### CODE REVIEW

File: _posts/2020-01-30-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+We've been reassessing this meeting and have made some changes. Introducing Origami Office Hours! We'll be hosting an hour-long session weekly but from now on it will be in the 3rd Floor breakout area. The agenda will be public and editable, and the session will be split in two: the first part will be dedicated to helping people who prefer to speak to the team at a set time, the second part will be a backlog grooming session for us which anyone is free to attend and contribute to.
+
+We'll be setting up invites soon and hope to see some of you there! Keep an eye on Slack and your calendars (also feel free to respond to this email and ask to be included).
+
+
+### Major Cascade
+
+At the end of last year a lot of Origami's time was spent working in Customer Products on the Major Cascade. This is where Origami release new major versions for each of our components, and it has an impact on teams using these components (for those interested in what a Major Cascade is and why we were doing it, [Lee wrote a blog post](http://localhost:4000/blog/2019/10/31/major-cascade/)).
```
Comment (notlee) [inline _posts/2020-01-30-newsletter.md]:
```suggestion
At the end of last year a lot of Origami's time was spent working in Customer Products on the Major Cascade. This is where Origami release new major versions for each of our components, and it has an impact on teams using these components (for those interested in what a Major Cascade is and why we were doing it, [Lee wrote a blog post](https://origami.ft.com/blog/2019/10/31/major-cascade/)).
```

File: _posts/2020-01-30-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  - [#origami-standup](https://app.slack.com/client/T025C95MN/CT0HFCKQX) for those who are _super_ interested in what we're doing day-to-day, this is where we share our morning update. Probably too much information for most, but we're doing this publicly now to be more transparent
+
+
+### Origami Office Hours (ooh?)
+
+The Origami Open Meeting has been around since the team originally formed (I couldn't find the exact date for the first one, but it was in 2013). One of its original purposes was to discuss Origami's progress with people from other areas of Product and Tech, but it has slowly evolved into more of a support forum.
+
+We've been reassessing this meeting and have made some changes. Introducing Origami Office Hours! We'll be hosting an hour-long session weekly but from now on it will be in the 3rd Floor breakout area. The agenda will be public and editable, and the session will be split in two: the first part will be dedicated to helping people who prefer to speak to the team at a set time, the second part will be a backlog grooming session for us which anyone is free to attend and contribute to.
```
Comment (JakeChampion) [inline _posts/2020-01-30-newsletter.md]: We should mention it is in Bracken House and can be joined via a hangout link (we should use the same hangout link each week)

**Comment by notlee** [comment]
This is a great update 🙌

File: _posts/2020-01-30-newsletter.md (hunk 1 above)
Comment (JakeChampion) [inline _posts/2020-01-30-newsletter.md]: that link points to locahost, we need to make sure it points to the correct place before publishing this

File: _posts/2020-01-30-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+
+### Major Cascade
+
+At the end of last year a lot of Origami's time was spent working in Customer Products on the Major Cascade. This is where Origami release new major versions for each of our components, and it has an impact on teams using these components (for those interested in what a Major Cascade is and why we were doing it, [Lee wrote a blog post](http://localhost:4000/blog/2019/10/31/major-cascade/)).
+
+Now that the work to migrate Customer Products is largely finished, we're available to help out in other areas of Product and Tech for anyone who needs it. If you're having difficulty migrating, then please give us a shout and we'll arrange something.
+
+
+## Special thanks
```
Comment (JakeChampion) [inline _posts/2020-01-30-newsletter.md]: Maybe we should also thank the people who have attended the origami workshops and given us feedback on them

File: _posts/2020-01-30-newsletter.md (hunk 1 above)
Comment (rowanmanning) [reply _posts/2020-01-30-newsletter.md]: Fixed by Lee 🙈

File: _posts/2020-01-30-newsletter.md (hunk 3 above)
Comment (rowanmanning) [reply _posts/2020-01-30-newsletter.md]: That's a good idea, I'll update.

File: _posts/2020-01-30-newsletter.md (hunk 2 above)
Comment (rowanmanning) [reply _posts/2020-01-30-newsletter.md]: Done, how is the updated version?

File: _posts/2020-01-30-newsletter.md (hunk 3 above)
Comment (rowanmanning) [reply _posts/2020-01-30-newsletter.md]: Done

**Pull request merged**